backend/MoleculeOSSite/Services/AccountService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+using MoleculeOSSite.Entities;
+using MoleculeOSSite.ModelsDTO;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace MoleculeOSSite.Services
+{
+
+    public interface IAccountService
+    {
+        void RegisterUser(User user);
+        string GenerateJwt(LoginDTO dto);
+    }
+
+    public class AccountService : IAccountService
+    {
+        private readonly MyDbContext _context;
+        private readonly IPasswordHasher<User> _passwordHasher;
+        private readonly AuthenticationSettings _authenticationSettings;
+
+        public AccountService(MyDbContext context, IPasswordHasher<User> passwordHasher,AuthenticationSettings authenticationSettings)
+        {
+            _context = context;
+            _passwordHasher = passwordHasher;
+            _authenticationSettings = authenticationSettings;
+        }
+
+        public string GenerateJwt(LoginDTO dto)
```

Comment:
```suggestion
        public string GenerateJwt(LoginDTO loginDto)
```
Nie ryzykowałbym używania takich skrótowców w tak dużej metodzie.